gulpfile.babel.js

```diff
@@ -39,11 +39,11 @@ const $ = gulpLoadPlugins();
 const reload = browserSync.reload;
 
 // Lint JavaScript
-gulp.task('jshint', () =>
+gulp.task('eslint', () =>
```

Comment:
`lint` seems more reasonable to me. If we add in other linters, then we can create sub-tasks for those and have the main `lint` task run them all.